server/server.go

```diff
@@ -232,29 +238,28 @@ func (s *server) Ping(context.Context, *rpcpb.PingRequest) (*rpcpb.PingResponse,
 	return &rpcpb.PingResponse{Pid: int32(os.Getpid())}, nil
 }
 
-const defaultStartTimeout = 5 * time.Minute
+func (s *server) Start(_ context.Context, req *rpcpb.StartRequest) (*rpcpb.StartResponse, error) {
+	s.mu.Lock()
+	defer s.mu.Unlock()
 
-func (s *server) Start(ctx context.Context, req *rpcpb.StartRequest) (*rpcpb.StartResponse, error) {
-	// if timeout is too small or not set, default to 5-min
-	if deadline, ok := ctx.Deadline(); !ok || time.Until(deadline) < defaultStartTimeout {
-		var cancel context.CancelFunc
-		ctx, cancel = context.WithTimeout(context.Background(), defaultStartTimeout)
-		_ = cancel // don't call since "start" is async, "curl" may not specify timeout
-		s.log.Info("received start request with default timeout", zap.String("timeout", defaultStartTimeout.String()))
-	} else {
-		s.log.Info("received start request with existing timeout", zap.String("timeout", deadline.String()))
+	// If [network] is already populated, the network has already been started.
+	if s.network != nil {
+		return nil, ErrNetworkRunning
```

Comment:
can use old naming? applies other places also

Reply:
The old naming was misleading -- it said "not bootstrapped" but it really meant "not running"

Reply:
https://github.com/ava-labs/avalanche-network-runner/pull/447#discussion_r1120182965